data/json/items/armor/suits_protection.json

```diff
@@ -846,6 +846,92 @@
     "proportional": { "weight": 1.125, "volume": 1.13, "price": 1.25 },
     "flags": [ "VARSIZE", "WATER_FRIENDLY", "SKINTIGHT", "STURDY", "OVERSIZE" ]
   },
+  {
+	"id": "officer_uniform",
+	"repairs_like": "trenchcoat",
+	"type": "ARMOR",
+	"name": {
+		"str": "Dress uniform"
+	},
+	"description": "A military officer's dress uniform from before the Cataclysm, complete with rank and ribbons. Just looking at it gives you an air of authority. Perhaps not the most practical military wear, but a very stylish option",
+	"weight": "1653 g",
+	"volume": "4750 ml",
```

Comment:
Keep your PR small and focused, this is _very important_, it makes it easier to review, this uniform and the "combo cover" can be added in a different pull request, they are not aftershock content so they should not be in an aftershock pull request.

Reply:
Yeah that was an accident sorry, I changed it now